public/index.html

```diff
@@ -29,6 +29,19 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>zOS</title>
+    <script>
+      // Copied from embedly documentation
+      (function(w, d){
+        var id='embedly-platform', n = 'script';
+        if (!d.getElementById(id)){
+            w.embedly = w.embedly || function() {(w.embedly.q = w.embedly.q || []).push(arguments);};
+            var e = d.createElement(n); e.id = id; e.async=1;
+            e.src = ('https:' === document.location.protocol ? 'https' : 'http') + '://cdn.embedly.com/widgets/platform.js';
+            var s = d.getElementsByTagName(n)[0];
+            s.parentNode.insertBefore(e, s);
+        }
+      })(window, document);
+    </script>
```

Comment:
do we need this? i don't think we should. getting rid of this stuff is the main reason we switched to the oembed version of their api.